docs/source/api/stats.rst

```diff
@@ -5,4 +5,21 @@ Statistics and diagnostics are delegated to the
 `ArviZ <https://arviz-devs.github.io/arviz/index.html>`_.
 library, a general purpose library for
 "exploratory analysis of Bayesian models."
-Refer to its documentation to use the diagnostics functions directly.
+For statistics and diagnostics, ``pymc3.<function>`` are now aliases
+for ArviZ functions. Thus, the links below will redirect you to
+ArviZ docs:
+
+.. currentmodule:: pymc3.stats
+
+
+- :func:`pymc3.bfmi <arviz:arviz.bfmi>`
+- :func:`pymc3.compare <arviz:arviz.compare>`
+- :func:`pymc3.ess <arviz:arviz.ess>`
+- :data:`pymc3.geweke <arviz:arviz.geweke>`
+- :func:`pymc3.hpd <arviz:arviz.hpd>`
```

Comment:
I understand wanting to keep the original `pm.hpd` name, but `az.hpd` no longer exists either, it is now `az.hdi`. This one is used and had a deprecation warning for several arviz releases indicating users to switch from hpd to hdi.

Note that currently the code does not do any hpd-hdi aliasing on pymc3 side though.